virtualization/Docker/Dockerfile.dev

```diff
@@ -0,0 +1,55 @@
+# Dockerfile for development
+# Based on the production Dockerfile, but with development additions.
+# Keep this file as close as possible to the production Dockerfile, so the environments match.
+
+FROM python:3.5
```

Comment:
I only changed the development Docker to match the production version. That was already changed to 3.5.